subsys/suit/cache/src/suit_dfu_cache_helpers.c

```diff
@@ -15,8 +15,33 @@
 #include "suit_ram_sink.h"
 #include "zcbor_noncanonical_decode.h"
 
+#ifdef CONFIG_FLASH_IPUC
+#include <drivers/flash/flash_ipuc.h>
+#endif /* CONFIG_FLASH_IPUC */
+
 LOG_MODULE_REGISTER(dfu_cache_helpers, CONFIG_SUIT_LOG_LEVEL);
 
+/* This function returns true if the given cache pool is an IPUC-based cache
+ * but is not initialized yet.
+ */
+static bool is_cache_ipuc_uninitialized(struct dfu_cache_pool *cache_pool)
```

Comment:
Please put the contents of this function under `#ifdef CONFIG_FLASH_IPUC`

Reply:
Done